### PR TITLE
[Task] Thumbnail for versions

### LINKED ISSUE #361: [Versions] Preview image thumbnails
The version preview images need to be thumbnails:

image

Currently it seems that the full image content get's displayed. This works for the small assets in the demo but will fail for big assets (or even strange formats like *.psd).

Related to #349

### PR DESCRIPTION
**Pull request by mattamon**
## Changes in this pull request
Resolves https://github.com/pimcore/studio-ui-bundle/issues/361

### CODE REVIEW

File: src/Version/Service/VersionBinaryService.php [hunk 1]
```diff
@@ -65,21 +67,29 @@ public function downloadAsset(
...
         $version = $this->repository->getVersionById($id);
-        $element = $this->repository->getElementFromVersion($version, $user);
-        if (!$element instanceof Asset\Image) {
-            throw new InvalidElementTypeException($element->getType());
+        $image = $this->repository->getElementFromVersion($version, $user);
+        if (!$image instanceof Asset\Image) {
+            throw new InvalidElementTypeException($image->getType());
+        }
+
+        $config = Config::getPreviewConfig();
```
Comment (lukmzig) [inline src/Version/Service/VersionBinaryService.php]: Should we introduce a new resolver here? Or you think its not necessary?

Comment (mattamon) [reply src/Version/Service/VersionBinaryService.php]: Ah yes, totally forgot about that for the review!
I will add a new resolver

**Review by lukmzig — COMMENTED** [review]
In general it looks good to me :)
One question, should we also change the download endpoint to return only asset with the pre-configured thumbnail? I am not sure if we need to keep it consistent

**Comment by mattamon** [comment]
> In general it looks good to me :) One question, should we also change the download endpoint to return only asset with the pre-configured thumbnail? I am not sure if we need to keep it consistent

Good question. I am also not sure about it. Maybe create an issue to keep that in mind.

**Comment by lukmzig** [comment]
> > In general it looks good to me :) One question, should we also change the download endpoint to return only asset with the pre-configured thumbnail? I am not sure if we need to keep it consistent
> 
> Good question. I am also not sure about it. Maybe create an issue to keep that in mind.

New issue is here: https://github.com/pimcore/studio-backend-bundle/issues/170

**Review by lukmzig — APPROVED** [review]
🦭

**Pull request merged**